### PR TITLE
Added Packetizer_init function

### PR DESCRIPTION
**Pull request by Glamhoth**

### CODE REVIEW

File: src/Packetizer/Packetizer.h [hunk 1]
```diff
@@ -54,6 +54,15 @@ typedef struct {
     uint16_t packetSequenceCount; ///< Counter for created packets
 } Packetizer;
 
+/**
+ * @brief   Initialize packetizer struct
+ *
+ * Initialized packet sequence count to 0
```
Comment (Lurkerpas) [inline src/Packetizer/Packetizer.h:60]: Initializes?

File: src/Packetizer/Packetizer.h [hunk 2]
```diff
@@ -54,6 +54,15 @@ typedef struct {
     uint16_t packetSequenceCount; ///< Counter for created packets
 } Packetizer;
 
+/**
+ * @brief   Initialize packetizer struct
```
Comment (Lurkerpas) [inline src/Packetizer/Packetizer.h:58]: Packetizer

File: src/Packetizer/Packetizer.h (hunk 1 above)
Comment (Glamhoth) [reply src/Packetizer/Packetizer.h]: Fixed

File: src/Packetizer/Packetizer.h (hunk 2 above)
Comment (Glamhoth) [reply src/Packetizer/Packetizer.h]: Fixed

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**